waspc/cli/Command/Common.hs

```diff
@@ -58,3 +73,10 @@ alphaWarningMessage =
   ( "NOTE: Wasp is still in Alpha, therefore not yet production ready "
       ++ "and might change significantly in the future versions."
   )
+
+notFoundError :: CommandError
```

Comment:
You are in Command.Common module, and since name of this function is just `notFoundError`, it is not specific enough!

I would advise renaming it to `waspProjectRootNotFoundError`.